controller/open.go

```diff
@@ -0,0 +1,73 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/railwayapp/cli/constants"
+	"github.com/railwayapp/cli/errors"
+)
+
+// OpenInBrowser opens the provided url in the browser
+func (c *Controller) OpenInBrowser(ctx context.Context, args []string, projectId string) error {
+	if len(args) == 0 {
+		nameList(projectId)
+		return nil
+	}
+	if url, ok := constants.DocsURLMap[args[0]]; ok {
+		if strings.Contains(url, "%s") {
+			url = fmt.Sprintf(url, projectId)
+		}
+		err := c.gtwy.OpenInBrowser(url)
+		if err != nil {
+			return err
+		}
+	} else {
+		return errors.OpenCommandNotFound
+	}
+	return nil
+
+}
+
+func nameList(id string) error {
+	names, longest := getNames()
+	fmt.Printf("%s%s\n", padName("shortcut", longest), "    url")
```

Comment:
I'm not sure its is clear that the shortcuts should be used with railway open

Maybe 

instead of just "docs" and "project" it is "railway open docs" and "railway open project"?